src/internal/methods/NeoArmMethod.h

```diff
@@ -221,7 +224,13 @@ template<typename T_SPEEDPROPS> class NeoArmMk20dxSpeedBase
                 while (ARM_DWT_CYCCNT - cyc < T_SPEEDPROPS::Cycles);
 
                 cyc = ARM_DWT_CYCCNT;
+
+                #if defined(KINETIS)
                 *set = 1;
+                #else
+                *set = msk;
```

Comment:
with the above this is just 
`*set = msk;`

Reply:
ok, done